README.rst

```diff
@@ -61,6 +62,9 @@ Create the client interface...
     >>> from pyhorn import MHClient
     >>> client = MHClient('http://matterhorn.example.edu', 'user', 'passwd')
```

Comment:
this is the rest endpoint username / password, right? Maybe that's totally obvious and not worth mentioning.